functional-tests/envs/testenv.py

```diff
@@ -227,6 +227,70 @@ def make_drt(self, el_address, musig_bridge_pk):
         return drt_tx_id
 
 
+class CrashTestBase(StrataTester):
+    """
+    Testbase for Crash Test
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        ctx.set_env("basic")
+
+    def main(self, ctx: flexitest.RunContext):
+        self.seq = ctx.get_service("sequencer")
+        self.seqrpc = self.seq.create_rpc()
+
+        self.debug("checking connectivity")
+        protocol_version = self.seqrpc.strata_protocolVersion()
+        assert protocol_version is not None, "Sequencer RPC inactive"
+
+        self.check_handle_bailout()
+
+        return True
+
+    def get_bail_tag(self) -> str:
+        raise NotImplementedError
+
+    def check_handle_bailout(self):
+        """
+        Handles the bailout process for the given sequencer RPC.
+
+        Raises:
+            AssertionError: If the bailout or chain tip progress fails.
+        """
+        # wait for 2 seconds for chain tip slot to accumulate.
+        # Since the chain tip requirement is not exact, we can sleep here
+        time.sleep(2)
```

Comment:
This should be our guideline, if you cannot do `wait_until` & Co. you need to add a comment explaining the why of `time.sleep`.